### PR TITLE
test(e2e): query params and selected token

### PR DESCRIPTION
**Pull request by 0xArdy**
test: 
- [x] query params
- [x] change seleted token
- [x] activate expert mode
- [x] doesnt show tokens with custom bridges (wstETH, SNX, DAI)
- [x] reset chains in expert mode

### CODE REVIEW

File: cypress/e2e/bridge.cy.ts [hunk 1]
```diff
@@ -82,4 +92,166 @@ describe('Brid.gg interaction tests', () => {
...
+    // Test case for checking if the 'amount' query parameter updates the token amount input to '1'
+    cy.visit('/op-mainnet?amount=1');
+    cy.getDataTest('token-amount-input').get('input').should('have.value', '1');
+
+    // Test case for updating the 'amount' query parameter to '2' and verifying the input value
+    cy.visit('/op-mainnet?amount=2');
+    cy.getDataTest('token-amount-input').get('input').should('have.value', '2');
+
+    // Test case for a decimal 'amount' query parameter and verifying the input reflects the exact value
+    cy.visit('/op-mainnet?amount=1.234');
```
Comment (0xShaito) [inline cypress/e2e/bridge.cy.ts:106]: Can we try 18 decimals?

File: cypress/e2e/bridge.cy.ts [hunk 2]
```diff
@@ -82,4 +92,166 @@ describe('Brid.gg interaction tests', () => {
...
+    cy.getDataTest('token-amount-input').get('input').should('have.value', '1.234');
+  });
+
+  it('handle the `token` query', () => {
+    // Test case for selecting the 'AAVE' token via the 'token' query parameter
+    cy.visit('/op-mainnet?token=AAVE');
+    cy.getDataTest('selected-token-button').should('have.text', 'AAVE');
+
+    // Test case for changing the selected token to 'USDC' through the 'token' query parameter
+    cy.visit('/op-mainnet?token=USDC');
```
Comment (0xShaito) [inline cypress/e2e/bridge.cy.ts:116]: Why do we have the query param as token name instead of address?

File: cypress/e2e/bridge.cy.ts [hunk 3]
```diff
@@ -82,4 +92,166 @@ describe('Brid.gg interaction tests', () => {
...
+    // Verify initial selected token
+    cy.getDataTest('selected-token-button').should('have.text', 'ETH');
+
+    // Change the selected token and verify
+    cy.getDataTest('selected-token-button').click();
+    cy.getDataTest('token-search-input')
+      .get('input[placeholder="Search name or paste address"]')
+      .type('usdc', { force: true });
+    cy.getDataTest('USDC-token').click();
+    cy.getDataTest('selected-token-button').should('have.text', 'USDC');
```
Comment (0xShaito) [inline cypress/e2e/bridge.cy.ts:143]: This changes the query param too right? Can we add that to the test?

**Review by 0xShaito — CHANGES_REQUESTED** [review]

File: cypress/e2e/bridge.cy.ts (hunk 2 above)
Comment (0xArdy) [reply cypress/e2e/bridge.cy.ts:116]: more user friendly, also we'll only have a short list of tokens

File: cypress/e2e/bridge.cy.ts (hunk 1 above)
Comment (0xArdy) [reply cypress/e2e/bridge.cy.ts:106]: done! https://github.com/defi-wonderland/superchain-bridge-ui/pull/129/commits/5f431cb10441f3cb7c1ed35db4490349ae5bc146

File: cypress/e2e/bridge.cy.ts (hunk 3 above)
Comment (0xArdy) [reply cypress/e2e/bridge.cy.ts:149]: done! https://github.com/defi-wonderland/superchain-bridge-ui/pull/129/commits/5f431cb10441f3cb7c1ed35db4490349ae5bc146

**Review by 0xShaito — APPROVED** [review]

**Review by 0xtiti — APPROVED** [review]

**Pull request merged**